cartridge/webui/gql-boxinfo.lua

```diff
@@ -195,7 +195,7 @@ local boxinfo_schema = {
                             ' replication log sequence numbers',
                     },
                     replication_info = {
-                        kind = gql_types.list(gql_replica_status.nonNull),
```

Comment:
To be precise it's not an array - actually it's a map where key is replica id, it's not abstract sparsed array

Reply:
Does graphql have such type?

Reply:
No, there is no "map" type. Maybe it should be `[{id: id, ...gql_replica_status}]`.
But ok, maybe we should preserve backward compatibility and stay it as is.